lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -1322,6 +1297,7 @@ def deploy_vm(self):
         self.configure_disks(vm_obj=vm_obj)
         self.configure_network(vm_obj=vm_obj)
         if self.should_deploy_from_template() and len(self.params['customizations']) > 0:
```

Comment:
To be honest I wasn't sure what my original code was after a merge-glitch during rebase (as it wasn't changed in my production code). So I put back what was there originally, not what I changed it to previously...